pkgs/top-level/linux-kernels.nix

```diff
@@ -279,8 +257,6 @@ in {
     linux_5_15_hardened = hardenedKernelFor kernels.linux_5_15 { };
     linux_6_1_hardened = hardenedKernelFor kernels.linux_6_1 { };
     linux_6_6_hardened = hardenedKernelFor kernels.linux_6_6 { };
-    linux_6_8_hardened = hardenedKernelFor kernels.linux_6_8 { };
-    linux_6_9_hardened = hardenedKernelFor kernels.linux_6_9 { };
```

Comment:
I'm sorry I didn't notice it earlier, but for the _hardened we also have `throws` in this file that are still missing for 6.8/6.9.

Reply:
No worries, fixed!